src/test/java/org/opensearch/knn/plugin/script/PainlessScriptIT.java

```diff
@@ -28,7 +28,7 @@
 import java.util.Map;
 import java.util.Objects;
 
-public class PainlessScriptScoringIT extends KNNRestTestCase {
+public class PainlessScriptIT extends KNNRestTestCase {
```

Comment:
Why we renaming this class?

Reply:
PainlessScriptScoringIT refers to that this test case only contains ScriptScore Context. I want to include additional context , hence, i renamed to PainlessScript to accommodate additional context's test too.